samples/node/connector.js

```diff
@@ -24,16 +24,23 @@ app.use(bodyParser.urlencoded());
 app.set('trust proxy', true);
 app.use(['/cards/requests', '/reports'], function (req, res, next) {
     const authorization = req.header("authorization");
+    const xAuthorization = req.header("X-Connector-Authorization");
     if (authorization) {
         console.log(`Client passed "${authorization}". We should authenticate using a public key from VMware Identity Manager`);
-        next();
     } else {
         res.status(401).send("Missing Authorization header");
+        return
+    }
+    if (xAuthorization) {
+        console.log(`Client passed "${xAuthorization}". Connector will use this to fetch info. form the backend Weather system.`);
+        next();
+    } else {
+        res.status(400).send("Missing X-Connector-Authorization header");
```

Comment:
Change this to:

```
const r = res.status(400);
r.setHeader('X-Backend-Status', 401);
return r.send('...');
```
So it will send the X-Backend-Status like the spec expects.